### ISSUE TITLE
Build instructions for macOS

### ISSUE THREAD

**Issue by elsurudo** [XQF/xqf#189]
I'm sure I'm in the minority trying to use this on macOS, but it would be nice!

**Comment by elsurudo** [comment]
Also, a pre-built binary as well

**Comment by illwieckz** [comment]
If you have a mac, you're by far the best one to find these instructions and to share them ;-)

**Comment by elsurudo** [comment]
Fair enough. Actually I have tried and gotten past the `cmake` step by installing all the dependencies. Now, the issue I have is that when I run `make`, the `games.c` file is not found – where does this file come from?

**Comment by illwieckz** [comment]
It's expected to be generated at build time (and yes it's ugly, the code compiles a code to run a binary to produces a source that will be compiled after that, that's the kind of weird stuff you can find in a 20 years old project :scream: ).

Do you have a `gamesxml2c` binary in your `build/` dir after having done the `cmake .. ; make` dance?

**Comment by elsurudo** [comment]
I had to do a `make gamesxml2c` between `cmake` and `make`, and I eventually got the binary to build. Still haven't gotten it to run successfully though – it can't find some dependencies. I think it's looking in the wrong places, and I will need to recompile with some flags set.
